src/components/exercises/Splash.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+import { View, Text, Dimensions } from 'react-native';
+import { RoundImageWithBorder} from '../Components';
+import { Font } from 'expo';
+import { BLACK,PRIMARY } from '../../styles/colors';
+import {DEFAULT} from'../../styles/text';
+
+class Splash extends Component {
+	render() {
+		return(
+		<View
+	        style={{
+          flex: 1,
+          backgroundColor: PRIMARY,
+          alignItems: 'center',
+          justifyContent: 'space-around'
+        }}>
+	 
+	        <RoundImageWithBorder
+	          image={this.props.images}
```

Comment:
I would inline images since we won't use the splash screen with another image. So instead of passing the image down, require it here directly.